README.md

```diff
@@ -220,6 +214,6 @@ DEBUG::      Processing line: bne     bimm12hi rs1 rs2 bimm12lo 14..12=1 6..2=0x
 ## How do I find where an instruction is defined?
 
 You can use `grep "^\s*<instr-name>" rv* unratified/rv*` OR run `make` and open
-`instr_dict.yaml` and search of the instruction you are looking for. Within that
+`instr_dict.json` and search of the instruction you are looking for. Within that
```

Comment:
@Timmmm I think the above can be rewritten like:

_" `instr_dict.json` and **search for** the instruction you are looking for. Within that ... "_

Reply:
@aswaterman I think these are also being pushed #304